force-app/main/default/lwc/viewNearbyEventsLWC/__tests__/viewNearbyEventsLWC.test.js

```diff
@@ -0,0 +1,86 @@
+import { createElement } from 'lwc';
+import ViewNearbyEventsLWC from 'c/viewNearbyEventsLWC';
+
+describe('c-view-nearby-events-l-w-c', () => {
+    // Helper function to wait until the microtask queue is empty. This is needed for promise
+    // timing when calling imperative Apex.
+    const flushPromises = () => new Promise(process.nextTick);
+
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('Test button tab', async () => {
+        const element = createElement('c-view-nearby-events-l-w-c', {
+            is: ViewNearbyEventsLWC
+        });
+
+        document.body.appendChild(element);
+
+        const addButton = element.shadowRoot.querySelector('button[data-id="goToExternalSite"]');
+        addButton.click();
+
+        await flushPromises();
+
+        const buttons = element.shadowRoot.querySelectorAll('button');
+        // Assert
+        expect(Array.from(buttons)).toContain(addButton);
```

Comment:
this assert does not make any sense because it is implied that the addButton exist
since you are clicking it
instead you should check that either the cancel or the continue button is defined